Sources/MapboxCoreNavigation/BundleAdditions.swift

```diff
@@ -27,7 +27,13 @@ extension Bundle {
      The Mapbox Core Navigation framework bundle.
      */
     public class var mapboxCoreNavigation: Bundle {
-        return Bundle(for: RouteController.self)
+        get {
+            #if SWIFT_PACKAGE
+            return .module
```

Comment:
Also use `Bundle.module` in the fallback at the bottom of `mapboxNavigationIfInstalled` below.

Reply:
Thanks for the suggestion, I'll fix this part.

Reply:
Because the `NSBundle.value(forKey: "mapboxNavigation")` is not working, and the `Bundle.module` would return the current module, I choose to filter the `bundleIdentifier` for current all bundles and to get the `MapboxNavigation` bundle.